cgi-bin/DW/Logic/ProfilePage.pm

```diff
@@ -851,6 +851,17 @@ sub external_services {
         };
     }
 
+    if ( my $medium = $u->prop( 'medium' ) ) {
+	my $medium = LJ::eurl( $medium );
```

Comment:
This should be `my $emedium = ...` so as not to overwrite `$medium`. Else, you end up with the content being double-escaped: eurl'd and then ehtml'd in the `text` value later.

So, change this one to `my $emedium ...` and then later use `$emedium` in the `url` value.